Makefile

```diff
@@ -762,6 +762,11 @@ test-azure-ipam: ## run the unit test for azure-ipam
 kind:
 	kind create cluster --config ./test/kind/kind.yaml
 
+# install azure Linux CNS and CNI dropgz images
+install-azure-images:
```

Comment:
if we're going to create standalone tooling for installing our components, let's do it correctly by writing some [small kustomize payloads](https://kustomize.io/) for them instead of hacking it with the integration test like this.